### DISCUSSION TITLE
Issues with Activating Copilot Subscription in VS Code

### DISCUSSION THREAD

**Discussion by umut-from-datajiro** [community/community#68998] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

I purchased a GitHub Copilot subscription but am unable to access the service in Visual Studio Code. Despite receiving a celebration screen indicating a successful transaction, my membership doesn't seem to have started. 

I've tried restarting VS Code and repurchasing several times with no success. I had previously used up the trial period. 

<img width="645" alt="image" src="https://github.com/community/community/assets/121491434/72ce3b4f-1a1a-4579-b58d-45e329717e66">

Has anyone else experienced this issue or have any suggestions on how to resolve it? Any insights would be appreciated. Thanks in advance!

**Comment by Akash1134** [comment, marked as answer]
Hi @umut-from-datajiro  The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
